### PR TITLE
Feature/r2.7.0 update docs

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for: https://issues.cask.co/browse/CDAP-1307

Staged at: [Installation](http://stg-web101.sw.joyent.continuuity.net/cdap/2.7.0-r2.7.0_update_docs/en/admin-manual/installation/installation.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -206,6 +206,10 @@ create a top-level ``/cdap`` directory in HDFS, owned by an HDFS user ``yarn``::
 
   sudo -u hdfs hadoop fs -mkdir /cdap && hadoop fs -chown yarn /cdap
 
+When running on a secure HBase cluster, as the ``hbase`` user, issue the command::
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Should this be under the _install-secure-hadoop section instead?

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 2]
```diff
@@ -428,6 +433,11 @@ In order to configure CDAP Master for Kerberos authentication:
       process. The string "_HOST" will be substituted with the local hostname.</description>
     </property>
 
+- The ``/cdap`` directory needs to be owned by the ``<cdap-principal>``; you can set
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Same as with the secure HBase command, we should frame this as "run as the hdfs user".  So maybe change the last part to: "you can set that by running the following command as the ``hdfs`` user (substituting for ``<cdap-principal>``)"

Then remove the "sudo -u hdfs" from the command.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 3]
```diff
@@ -569,4 +579,25 @@ and then restart CDAP.
...
+     
+If you are upgrading a secure Hadoop cluster, see the notes above about
+:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+
+.. _install-troubleshooting:
+
+Troubleshooting
+---------------------------------
+
+- If you are using a LinuxContainerExecutor, the ``cdap`` user needs to be present on all
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: "If you have YARN configured to use LinuxContainerExecutor (see the setting for ``yarn.nodemanager.container-executor.class``), ..."

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 4]
```diff
@@ -569,4 +579,25 @@ and then restart CDAP.
...
+  Hadoop nodes.
+
+- If you are using a LinuxContainerExecutor, and the UID for the ``cdap`` user is less than
+  500, you will need to add the ``cdap`` user to the allowed users configuration for the
+  LinuxContainerExecutor in Yarn by editing the ``/etc/hadoop/conf/container-executor.cfg``
+  file::
+
+    sudo vi /etc/hadoop/conf/container-executor.cfg
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: We can drop this line and assume they know how to edit the file.  Just leave the lines below.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Moved.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 4 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

**Comment by awholegunch** [comment]
Thanks for the review. Updates made and restaged. Please take another look.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 5]
```diff
@@ -428,6 +429,16 @@ In order to configure CDAP Master for Kerberos authentication:
       process. The string "_HOST" will be substituted with the local hostname.</description>
     </property>
 
+- The ``/cdap`` directory needs to be owned by the ``<cdap-principal>``; you can set
+  that by running the following command as the ``hbase`` user (substituting for
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: s/hbase/hdfs/

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 6]
```diff
@@ -428,6 +429,16 @@ In order to configure CDAP Master for Kerberos authentication:
...
+  ``<cdap-principal>``)::
+  
+    hadoop fs -mkdir /cdap && hadoop fs -chown <cdap-principal> /cdap
+    
+- When running on a secure HBase cluster, as the ``hbase`` user, issue the command::
+
+    echo "grant 'cdap', 'ACRW'" | hbase shell
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Sorry should have caught this before, we should replace 'cdap' here with '<cdap-principal>' and repeat instructions to substitute the cdap prinicpal name.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 7]
```diff
@@ -569,4 +580,22 @@ and then restart CDAP.
...
+If you are upgrading a secure Hadoop cluster, see the notes above about
+:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+
+.. _install-troubleshooting:
+
+Troubleshooting
+---------------------------------
+
+- If you have YARN configured to use LinuxContainerExecutor (see the setting for
+  ``yarn.nodemanager.container-executor.class``), the ``cdap`` user needs to be present on
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: This "cdap" mention and the one below should technically be "<cdap-principal>" as well.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 5 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 6 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 8]
```diff
@@ -428,6 +429,18 @@ In order to configure CDAP Master for Kerberos authentication:
...
+  are free to use a different appropriate name.
+
+- The ``/cdap`` directory needs to be owned by the ``<cdap-principal>``; you can set
+  that by running the following command as the ``hdfs`` user::
+  
+    hadoop fs -mkdir /cdap && hadoop fs -chown cdap /cdap
+    
+- When running on a secure HBase cluster, as the ``hbase`` user, issue the command::
+
+    echo "grant cdap>, 'ACRW'" | hbase shell
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: should be "grant 'cdap',..."

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 9]
```diff
@@ -569,4 +582,22 @@ and then restart CDAP.
...
+- If you have YARN configured to use LinuxContainerExecutor (see the setting for
+  ``yarn.nodemanager.container-executor.class``), the ``cdap`` user needs to be present on
+  all Hadoop nodes.
+
+- If you are using a LinuxContainerExecutor, and the UID for the ``cdap`` user is less than
+  500, you will need to add the ``cdap`` user to the allowed users configuration for the
+  LinuxContainerExecutor in Yarn by editing the ``/etc/hadoop/conf/container-executor.cfg``
+  file. Change the line for ``allowed.system.users`` to::
 
+    allowed.system.users=yarn,cdap
```
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Actually make this just:

allowed.system.users=cdap

"yarn" is not normally listed there.  That is just another artifact of our old coopr templates running secure clusters as yarn.

**Comment by ghelmling** [comment]
lgtm

**Pull request merged**